kable-api-v1.yaml

```diff
@@ -1415,6 +1415,307 @@ paths:
+        "401":
+          description: "unauthorized"
+        "404":
+          description: "not found"
+
+  /bundles/create:
+    post:
+      tags:
+        - bundles
+      summary: create a bundle
+      operationId: createBundle
+      description: |
+        Create a bundle.
+      parameters:
+        - in: header
+          name: Kable-Client-Id
+          description: Your client ID, found in the dashboard of your Kable account.
+          schema:
+            type: string
+          example: kci_3c90e9ac92c64f31ae8ed84d21e18740
+          required: true
+        - in: header
+          name: Kable-Client-Secret
+          description: Your `LIVE` or `TEST` secret key. Customers exist across all environments, so it does not matter which environment you use to create customers. Each customer will have separate keys for `LIVE` and `TEST` environments of your API.
+          schema:
+            type: string
+          example: sk_test.jI92Cbu0.XeHWdYM1VTLy4oLtGMw8wrmpt5q9d04n
+          required: true
+      requestBody:
+        description: Information about the bundle to create.
+        content:
+          application/json:
+            schema:
+              type: object
+              required:
+                - name
+              properties:
+                name:
+                  type: string
+                  example: "Messages and Connections Bundle"
+                  description: "A human-readable name for the bundle, visible on dashboards, invoices, and reports."
+                externalId:
+                  type: string
+                  example: "yourcompany_bundle123"
+                  description: "An identifier for the bundle as defined by your API."
+                description:
+                  type: string
+                  example: "A usage bundle based on the number of messages and connections"
+                  description: "A human-readable description for the bundle, visible on dashboards, invoices, and reports."
+                nickname:
+                  type: string
+                  example: "messages_connections_v1"
+                  description: An additional identifier for the bundle, defined by you, that is *not* visible to customers. If you have different *Monthly Active Users Bundle*s for different cohorts of customers, this is a useful place to differentiate those.
+                planIds:
+                  type: array
+                  items:
+                    type: string
+                  description: "A collection of plans to include in the bundle."
+                  example: ["pln_a23e148584d0463482c961b7f62f824c"]
+                costPerCredit:
+                  type: number
+                  example: 1.2
+                  description: An additional identifier for the bundle, defined by you, that is *not* visible to customers. If you have different *Monthly Active Users Bundle*s for different cohorts of customers, this is a useful place to differentiate those.
```

Comment:
`description: "Overage costs per credit for bundle subscribers (only relevant if bundle plans are priced in credits).`